### PR TITLE
Clean up descripitions

### PR DESCRIPTION
**Pull request by dbartholomae**
- make descripionts more valuable
- remove duplicate entry from UI
- remove SecureAuth as its documentation is hard to find, making it less awesome
- remove Wix Booking as it can't really be used outside Wix
- remove vcita and Setster due to missing frontend integration
- remove BookitLive because you can only test it with a credit card which is not awesome
- remove GetSocial as it is only for mobile apps
- remove releasenotes.io, productfeed.app and headwayapp.co as the missing documentation makes it less awesome
- remove skyflow as it isn't frontend focussed
- remove Warfare Plugins as it is WordPress only
- remove ClickFunnels and CartFlows as they cannot be integrated into an app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,142 +38,124 @@
...
-- [Froala Design Blocks](https://froala.com/design-blocks/) - Over 170 responsive design blocks for web or mobile apps.
+- [UIfort](https://uifort.com/) - UI design systems for all major frontend frameworks.
+- [Sematic UI](https://semantic-ui.com/) - Open source UI component library for React, Meteor, Ember and Angular.
+- [Design Revision's Shards Pro](https://designrevision.com/downloads/shards-pro) - Design blocks for web or mobile apps.
 - [Themesberg](https://themesberg.com) - Bootstrap templates, themes and UI kits.
-- [Treact](https://treact.owaiskhan.me/) - React templates for every need.
+- [Treact](https://treact.owaiskhan.me/) - React templates for landing pages, built on Tailwind CSS.
 
 ## Exit Intent Popup
 - [Wisepops](https://wisepops.com/) - Create contextual popups, banners and bars for mobile and desktop visitors.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Wisepops](https://wisepops.com/) - Contextual popups, banners and bars for mobile and desktop visitors.
```

File: README.md [hunk 2]
```diff
@@ -40,142 +38,124 @@
...
-- [Listagram](https://www.listagram.com) - Interactive opt-in form in less than 30 minutes.
-- [Poptin](https://www.poptin.com) - Engaging popups and forms for websites in minutes.
-- [Sleeknote's Exit Intent Popup](https://sleeknote.com/exit-intent-popup) - Create high-converting exit-intent popups.
+- [Listagram](https://www.listagram.com) - Interactive opt-in form for collecting email addresses.
+- [Poptin](https://www.poptin.com) - Popups and forms that can be integrated anywhere in the website.
+- [Sleeknote's Exit Intent Popup](https://sleeknote.com/exit-intent-popup) - Create exit-intent popups.
```
Comment (thisismydesign) [inline README.md]: Something specific here?

File: README.md [hunk 3]
```diff
@@ -40,142 +38,124 @@
...
-- [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/#security) - Universal platform to manage and secure identities.
-- [AWS cognito](https://aws.amazon.com/cognito/) - User sign-up, sign-in, and access control.
-- [SecureAuth](https://www.secureauth.com/products/identity-platform) - Identity platform focused on security and user experience.
+- [Auth0](https://auth0.com/) - Authentication management with native support for Angular, React and VueJS.
+- [Firebase](https://firebase.google.com) - PaaS from Google, including authentication management.
+- [Okta](https://www.okta.com/) - Authentication and Authorization solution with support for Angular, React and VueJS.
+- [Onelogin](https://www.onelogin.com/) - Authentication focussed on company internal applications.
+- [Microsoft Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/#security) - Authentication management in Azure, based on Microsoft accounts.
+- [AWS Cognito](https://aws.amazon.com/cognito/) - AWS authentication management. Frontend integration is part of [AWS Amplify](https://aws.amazon.com/de/amplify/).
```
Comment (thisismydesign) [inline README.md:64]: Descriptions shouldn't include links imo.
```suggestion
- [AWS Cognito](https://aws.amazon.com/cognito/) - AWS authentication management. Frontend integration is part of AWS Amplify.
```

File: README.md [hunk 4]
```diff
@@ -40,142 +38,124 @@
...
 
 ## Notification
-- [Notific](https://notific.io/) - Powerful, complete notification system.
-- [OneSignal](https://onesignal.com/) - Push notifications delivered.
-- [WonderPush](https://www.wonderpush.com/) - Unlimited push notifications for iOS, Android and web.
-- [VWO Engage](https://vwo.com/engage/push-notifications/) - Engage lost visitors and drive them back to the website with personalized notifications.
-- [Signalize](https://signalize.com/) - Data-driven web and mobile push notifications.
+- [OneSignal](https://onesignal.com/) - Mobile and web notifications.
+- [Notific](https://notific.io/) - Notifications including a PHP SDK.
+- [WonderPush](https://www.wonderpush.com/) - Focusing on notification speed.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [WonderPush](https://www.wonderpush.com/) - Notifications focused on speed.
```

File: README.md [hunk 5]
```diff
@@ -40,142 +38,124 @@
...
 - [Cookiefirst](https://cookiefirst.com/) - Cookie consent manager.
-- [usercentrics](https://usercentrics.com/) - Compliance & marketing in harmony.
-- [Sourcepoint](https://www.sourcepoint.com/) - The privacy management software designed to improve experiences.
-- [PiwikPro](https://piwik.pro/gdpr-consent-manager/) - One place to manage the marketing stack's compliance.
+- [usercentrics](https://usercentrics.com/) - Compliance manager that can be implemented as part of existing pages.
+- [Sourcepoint](https://www.sourcepoint.com/) - Compliance management as part of a suite including also ad blocker recovery.
+- [PiwikPro](https://piwik.pro/gdpr-consent-manager/) - Compliance management as part of an analytics tool.
 - [Consent Management Provider](https://www.consentmanager.de/) - German management tool for Cookie Consent.
 - [UniConsent](https://www.uniconsent.com/) - Consent management platform for GDPR, CCPA, COPPA compliance.
 - [TrustArc](https://trustarc.com/) - Simplifies privacy management for the CCPA, GDPR, LGPD and 900+ other global regulations.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [TrustArc](https://trustarc.com/) - Privacy management for the CCPA, GDPR, LGPD and other global regulations.
```

File: README.md [hunk 6]
```diff
@@ -40,142 +38,124 @@
...
 - [Cookiebot](https://www.cookiebot.com/) - Cookie tracking manager.
-- [iubenda](https://www.iubenda.com/) - Make sites and apps compliant with the law.
-- [Civic Cookie Control](https://www.civicuk.com/cookie-control/) - Cookie and GDPR compliance.
+- [iubenda](https://www.iubenda.com/) - Consent management and terms and conditions generator.
+- [Civic Cookie Control](https://www.civicuk.com/cookie-control/) - Smaller Cookie and GDPR compliance product.
```
Comment (thisismydesign) [inline README.md:115]:
```suggestion
- [Civic Cookie Control](https://www.civicuk.com/cookie-control/) - Standalone Cookie and GDPR compliance product.
```

File: README.md [hunk 7]
```diff
@@ -40,142 +38,124 @@
...
 - [CCM19](https://www.ccm19.de/) - German cookie consent manager.  
-- [CookieInformation](https://cookieinformation.com/) - Cookie compliance for DSGVO & ePrivacy.
-- [CookiePro](https://www.cookiepro.com/) - Compliance support across all major privacy regulations and frameworks.
+- [CookieInformation](https://cookieinformation.com/) - Focus on compliance reporting via a dashboard.
+- [CookiePro](https://www.cookiepro.com/) - Cookie management as part of a bigger privacy management solution.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [CookiePro](https://www.cookiepro.com/) - Cookie management as part of a privacy management solution.
```

File: README.md [hunk 8]
```diff
@@ -40,142 +38,124 @@
...
 
 ## Media
-- [Mux](https://mux.com/) - API-first platform, powered by data and designed by video experts to make video possible for every development team.
+- [Mux](https://mux.com/) - Stream hosted and live videos.
 - [Spotify web playback SDK](https://developer.spotify.com/documentation/web-playback-sdk/) - Client-side JavaScript library which allows you to create a new player in Spotify Connect and play any audio track from Spotify in the browser via encrypted media extensions.
 
 ## Form
-- [Static Forms](https://www.staticforms.xyz/) - HTML form for websites.
-- [Form King](https://www.formking.io/) - Fast and secure forms for static websites.
+- [Static Forms](https://www.staticforms.xyz/) - Get form results sent by email.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Static Forms](https://www.staticforms.xyz/) - Form that sends submissions by email.
```

File: README.md [hunk 9]
```diff
@@ -40,142 +38,124 @@
...
+- [Form King](https://www.formking.io/) - See form results in a dashboard.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Form King](https://www.formking.io/) - Form with an admin dashboard.
```

File: README.md [hunk 10]
```diff
@@ -40,142 +38,124 @@
...
 - [FlyingForms](https://flyingforms.io/?utm_source=github-awesome-components&utm_medium=referral&utm_campaign=awesome-list) - Building, versioning and managing forms.
 
 ## Marketing
-- [TrustPulse](https://trustpulse.com/) - Grow businesses with honest marketing.
+- [TrustPulse](https://trustpulse.com/) - Shows recent buyers on you website.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [TrustPulse](https://trustpulse.com/) - Shows recent buyers on the website.
```

File: README.md [hunk 11]
```diff
@@ -40,142 +38,124 @@
...
 
 ## Changelog
-- [Beamer](https://www.getbeamer.com) - Update and engage users effortlessly.
+- [Beamer](https://www.getbeamer.com) - Combination of push notifications and changelog management.
 - [Changelogfy](https://changelogfy.com/) - Changelogfy is a software and product release notes tool.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Changelogfy](https://changelogfy.com/) - Software and product release notes tool.
```

File: README.md [hunk 12]
```diff
@@ -40,142 +38,124 @@
...
-- [Changefeed](https://changefeed.app/) - Changelog for products in seconds.
-- [Release notes](https://releasenotes.io) - Craft engaging release notes to keep customers and stakeholders up to date, or as a marketing tool to convert more.
-- [Headway](https://headwayapp.co/) - Easiest way to keep customers in the loop about products.
-- [Canny's Changelog](https://canny.io/features/changelog) - Announce product updates.
-- [ProductFlare's Changelog](https://www.productflare.com/changelog) - Keep users updated, engaged and increase user retention with real-time product updates.
-- [ProductFeed](https://productfeed.app) - Retain users and keep them updated.
-- [Changes](https://changes.blue) - Keep users in the loop about product updates with a modern changelog that grows with you.
+- [Changefeed](https://changefeed.app/) - Set up a changelog page or Embed your changelog in your website.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Changefeed](https://changefeed.app/) - Changelog page and embedded widget on the website.
```

File: README.md [hunk 13]
```diff
@@ -40,142 +38,124 @@
...
+- [Canny's Changelog](https://canny.io/features/changelog) - Combines changelog with product roadmaps.
+- [ProductFlare's Changelog](https://www.productflare.com/changelog) - Changelog widget around a product team task management system.
+- [Changes](https://changes.blue) - Changelog building kit.
 
 ## Telecommunication
-- [Vonage](https://www.vonage.com/) - Flexible and scalable voice, messaging, video and data capabilities across Unified Communications, Contact Centers and Communications APIs.
-
-## Privacy
-- [Skyflow](https://www.skyflow.com/) - API for privacy.
+- [Vonage](https://www.vonage.com/) - Voice, messaging, video and data capabilities across Unified Communications, Contact Centers and Communications APIs.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Vonage](https://www.vonage.com/) - Voice, messaging, video and data capabilities across unified communications, contact centers and communication APIs.
```

File: README.md [hunk 14]
```diff
@@ -40,142 +38,124 @@
...
-- [ShareThis](https://sharethis.com/) - Grow audience and keep them engaged.
-- [AddToAny](https://www.addtoany.com/) - Universal sharing.
-- [WarefarePlugins](https://warfareplugins.com/) - Social sharing for WordPress.
-
-## Sales Funnel
-- [Click Funnels](https://www.clickfunnels.com/) - Create sales funnels that convert visitors into leads and then customers.
-- [CartFlows](https://cartflows.com/) - Get more leads, increase conversions, & maximize profits.
+- [AddThis](https://www.addthis.com/) - Social sharing buttons, available on AMPs.
+- [ShareThis](https://sharethis.com/) - Social sharing buttons.
+- [AddToAny](https://www.addtoany.com/) - Sharing buttons for ~100 different target services.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [AddToAny](https://www.addtoany.com/) - Sharing buttons for different target services.
```

File: README.md [hunk 15]
```diff
@@ -40,142 +38,124 @@
...
 
 ## Search
-- [Algolia](https://www.algolia.com/) - Make every search releavnt, immersive, optimized, engaing, intelligent, personalized, business-driven, etc.
+- [Algolia](https://www.algolia.com/) - Search for your application.
```
Comment (thisismydesign) [inline README.md:151]:
```suggestion
- [Algolia](https://www.algolia.com/) - Search for the whole application.
```

File: README.md [hunk 16]
```diff
@@ -40,142 +38,124 @@
...
 
 ## Feature flags
-- [Optimizely's Rollouts](https://www.optimizely.com/rollouts) - Ship faster and prove value with less risk.
-- [LaunchDarkly](https://launchdarkly.com/) - Manage features at any scale.
-- [Rollout](https://rollout.io/) - Progressively deliver features to users with confidence, in real-time.
-- [Split](https://www.split.io/) - Confidently release features as fast as you develop them.
+- [Optimizely's Rollouts](https://www.optimizely.com/rollouts) - Feature flags combined with A/B testing.
+- [LaunchDarkly](https://launchdarkly.com/) - Feature management with multiple integrations in CI servers and ticketing systems.
+- [Split](https://www.split.io/) - Another feature management with multiple integrations in CI servers and ticketing systems.
```
Comment (thisismydesign) [inline README.md:156]:
```suggestion
- [Split](https://www.split.io/) - Feature management with multiple integrations in CI servers and ticketing systems.
```

File: README.md [hunk 17]
```diff
@@ -40,142 +38,124 @@
...
+- [Rollout](https://rollout.io/) - Focus on segmented roll-out of features.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Rollout](https://rollout.io/) - Feature management focused on segmented roll-out of features.
```

**Review by thisismydesign — CHANGES_REQUESTED** [review]
In general "Create XYZ" descriptions are not description-like IMO. It could instead be "Tool to create XYZ".

Would avoid subjective words like "smaller" and "bigger".

Also avoid "your": https://github.com/componently-com/awesome-building-blocks-for-web-apps/blob/master/contributing.md#what

Would also avoid numbers, will get outdated.

^ I will add these to the contribution guidelines as well.

File: README.md (hunk 3 above)
Comment (dbartholomae) [reply README.md:64]: Why not? The link would be helpful and doesn't make reading harder.

File: README.md (hunk 6 above)
Comment (dbartholomae) [reply README.md:115]: Not sure what "standalone" is supposed to mean. This one is one of the smaller companies, that's why I put this into the description.

File: README.md (hunk 10 above)
Comment (dbartholomae) [reply README.md]: There's a typo in "your" which I fixed, but I would not got to "the" which I think is confusing.

File: README.md (hunk 14 above)
Comment (dbartholomae) [reply README.md]: The high number is what differentiates this solution from the others. I've updated to `Sharing buttons for many different target services.` to make the description even less brittle.

File: README.md (hunk 3 above)
Comment (thisismydesign) [reply README.md:64]: To simplify the format and to direct users to the relevant place. This way we also don't have to verify additional links.

File: README.md (hunk 15 above)
Comment (dbartholomae) [reply README.md:151]: Would also use "you" here as it is less sterile and confusing.

File: README.md (hunk 10 above)
Comment (thisismydesign) [reply README.md]: How about "Shows recent buyers on websites."? "Yours" I think is redundant.

File: README.md (hunk 16 above)
Comment (dbartholomae) [reply README.md:156]: That's the same description as the item above which looks like a copy-paste error. That's why I added "another" to make clear that it isn't.

File: README.md (hunk 6 above)
Comment (thisismydesign) [reply README.md:115]: Was thinking as opposed to "part of a bigger privacy management solution". But yea, standalone isn't great but I would skip "Smaller" nevertheless, feel free to skip both.

File: README.md (hunk 14 above)
Comment (thisismydesign) [reply README.md]: Cool!

File: README.md (hunk 15 above)
Comment (thisismydesign) [reply README.md:151]: > Search for applications?

Comment (dbartholomae) [reply README.md:151]: That's even more confusing. That sounds like you would search for applications

File: README.md (hunk 6 above)
Comment (dbartholomae) [reply README.md:115]: This is the main difference between this tool and the other, so I would rather leave the "smaller"

File: README.md (hunk 3 above)
Comment (dbartholomae) [reply README.md:64]: From my perspective the main link is obvious to anyone looking at the list, I don't see a danger of this link being confusing. And I also think that verifying additional helpful links actually will make the list even more useful.

Comment (thisismydesign) [reply README.md:64]: Another argument is consistency. We could link everything then, including frameworks. I don't really mind it, but trying to conform to other lists I saw.

Comment (dbartholomae) [reply README.md:64]: For me the difference is that for this one it is not really obvious that the frontend library is in a different part of AWS then the backend. I haven't encountered that problem for any of the other packages. It's basically working around AWS not-so-good documentation.

**Review by thisismydesign — APPROVED** [review]

**Pull request merged**